src/vendors/oasisscan.ts

```diff
@@ -22,6 +22,8 @@ import {
 
 import { throwAPIErrors } from './helpers'
 
+const getTransactionCacheMap: Record<string, Transaction> = {}
+
 export function getOasisscanAPIs(url: string | 'https://api.oasisscan.com/mainnet') {
```

Comment:
Hm, need to differentiate testnet from mainnet. You can add `url` to be part of the key

Reply:
Yes, good point.